build.gradle

```diff
@@ -34,6 +34,12 @@ subprojects {
     if (JavaVersion.current().isJava8Compatible()) {
         apply plugin: "net.ltgt.errorprone"
         apply plugin: 'com.github.sherter.google-java-format'
+
+        configurations.errorprone {
+            dependencies {
+                errorprone 'com.google.errorprone:error_prone_core:2.1.2'
```

Comment:
you have some versions of errorprone at lines: 126, 158. Please use a version variable and be consistent.

Also I think you can remove this and just update the one at line 158.

Reply:
I missed the version on line 158!  I was surprised that the Error Prone plugin wasn't using the latest version.  I updated that version number instead.